repose-aggregator/components/filters/client-auth/src/main/java/org/openrepose/filters/clientauth/common/AuthenticationHandler.java

```diff
@@ -96,10 +97,16 @@ protected AuthenticationHandler(Configurables configurables, AuthTokenCache cach
 
     protected abstract AuthToken validateToken(ExtractorResult<String> account, String token) throws AuthServiceException;
 
+    protected abstract AuthToken validateToken(ExtractorResult<String> account, String token, String requestGuid) throws AuthServiceException;
```

Comment:
The only reason would be backwards compatibility. Internally, I don't think we call the old methods anymore. In this instance, I thought it best to preserve the old interface for the time being simply because soon enough the entire filter will be re-written.

Excision of the old methods should be trivial.